packages/cli/src/auth/auth.service.ts

```diff
@@ -18,16 +18,19 @@ import { UrlService } from '@/services/url.service';
 interface AuthJwtPayload {
 	/** User Id */
 	id: string;
```

Comment:
I was doing that originally, but reverted that bit, and have moved that to another branch because I'd like to refactor a bit more of this code, and add additional tests.